src/dag/resolvers/mt/sorters/sorter_live.rs

```diff
@@ -606,7 +606,8 @@ impl<F: SmallField, Cfg: CSResolverConfig, RW: ResolutionRecordWriter> ResolverS
         for (i, item) in self.record.items[..].iter_mut().enumerate() {
             debug_assert_eq!(i, item.added_at as usize);
         }
-        self.record.values_count = 1 + unsafe { self.common.values.u_deref().max_tracked } as usize;
+
+        self.record.values_count = unsafe { self.common.values.u_deref().max_tracked + 1 } as usize;
```

Comment:
max_tracked is initialized as '-1'.
in the code before, it was then casted to usize --> which resulted in it being usize::max()
and then when we added '+1' - it overflowed

Reply:
There were (are) cases when the resolver is initialized with zero values, when it is unused. I've changed so that wouldn't actually happen, but it seems there's more places where this occurs, that are outside of harness and shivini test scopes.